### PR TITLE
Let's get roasted

### PR DESCRIPTION
**Pull request by bradleypurdue**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>
+			Bradley Purdue - Full Stack Developer
```
Comment (mporam) [inline index.html]: unnecessary line breaks?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+		</title>
+		
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		
+		<link rel="stylesheet" href="./style.css" />
+	</head>
+	<body>
+		<header>
```
Comment (mporam) [inline index.html]: entire file is tab indented, please use spaces

File: index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+							CONTACT ME
+						</a>
+					</li>
+				</ul>
+			</nav>
+			<div id="header-content">
+				<h1>
+					 Bradley Purdue
+				</h1>
+				<p>
```
Comment (mporam) [inline index.html]: this isnt really a paragraph, is a sub head, consider using h2 or h3

File: index.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+							<div class="portfolio-item-content">
+								<h3>
+									Pilot Shop: Website Recreation
+								</h3>
+								<p>
+									Using only the HTML and CSS skills that we had been taught throughout the first week of Mayden Academy,
+									we were challenged to re-create a static version of Shopify store with our knowledge.
+								</p>
+								<p>
+									<strong>
```
Comment (mporam) [inline index.html]: strong is used to emphasis text. unless this is important (its not), use CSS to make it bold

File: index.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>
+			Bradley Purdue - Full Stack Developer
+		</title>
+		
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+
```
Comment (mporam) [inline index.html]: unecessary line breaks?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+									</li>
+								</ul>
+							</div>
+						</article>
+					</div>
+				</div>
+			</section>
+		</main>
+		<footer>
+			<section id="contact">
```
Comment (mporam) [inline index.html]: unnecessary section tag

File: style.css [hunk 7]
```diff
@@ -0,0 +1,347 @@
...
+	display: inline;
+	
+	font-weight: bold;
+}
+
+header nav li:hover {
+	border-bottom: 2px solid var(--theme);
+}
+
+header #header-content {
```
Comment (mporam) [inline style.css]: doesnt need to be an ID, also doesnt need the word `header` in it, as its in the `header` tag

File: style.css [hunk 8]
```diff
@@ -0,0 +1,347 @@
...
+	}
+	
+	header h3 {
+		font-size: 2rem;
+	}
+}
+
+
+/* About */
+#about {
```
Comment (mporam) [inline style.css]: generally you never need an ID. try use classes wherever possible

File: style.css [hunk 9]
```diff
@@ -0,0 +1,347 @@
...
+
+#portfolio-items {
+	display: flex;
+	
+	justify-content: space-between;
+	flex-wrap: wrap;
+}
+
+.portfolio-item {
+	width: 100%;
```
Comment (mporam) [inline style.css]: this does nothing

File: style.css [hunk 10]
```diff
@@ -0,0 +1,347 @@
...
+	margin-bottom: 25px;
+	
+	display: flex;
+	
+	justify-content: space-between;
+}
+
+@media only screen and (max-width: 1000px) {
+	.portfolio-item {
+		width: 100%;
```
Comment (mporam) [inline style.css]: also does nothing

File: style.css [hunk 11]
```diff
@@ -0,0 +1,347 @@
...
+	}
+}
+
+.portfolio-item-preview {
+	width: 100%;
```
Comment (mporam) [inline style.css]: this does nothing if you also remove it from the `.portfolio-item-content` box

File: style.css [hunk 12]
```diff
@@ -0,0 +1,347 @@
...
+	padding: 50px;
+}
+
+#portfolio h2 {
+	margin: 50px;
+	
+	text-align: center;
+	
+	font-size: 3rem;
+	font-family: "Roboto Slap", serif;
```
Comment (mporam) [inline style.css]: looks like all your `h1`, `h2`, `h3` tags are this font, you can set this once at the top as:
`h1, h2, h3 {`...

File: style.css [hunk 13]
```diff
@@ -0,0 +1,347 @@
...
+
+	display: flex;
+	
+	justify-content: space-between;
+	
+	text-align: center;
+}
+
+.portfolio-item li {
+	width: 49%;
```
Comment (mporam) [inline style.css]: not a good solution for this. instead use padding to increase the button size

File: style.css [hunk 14]
```diff
@@ -0,0 +1,347 @@
...
+	
+	background-color :pink;
```
Comment (mporam) [inline style.css]: why?? and please dont use text based colors

File: style.css [hunk 15]
```diff
@@ -0,0 +1,347 @@
...
+	display: inline;
+}
+
+.portfolio-item a {
+	width: 100%;
```
Comment (mporam) [inline style.css]: this does nothing

File: style.css (hunk 14 above)
Comment (bradleypurdue) [reply style.css]: This was for testing and I forgot to remove it, my bad.

**Pull request merged**